scipy/signal/_spline_filters.py

```diff
@@ -644,3 +647,162 @@ def qspline1d_eval(cj, newx, dx=1.0, x0=0):
         result += cj[indj] * _quadratic(newx - thisj)
     res[cond3] = result
     return res
+
+
+def symiirorder1(signal, c0, z1, precision=-1.0):
```

Comment:
python functions `symiirorder1` and symiirorder2` are shown as new here but are in fact just moved from `_splines.py`,  where they were introduced in https://github.com/scipy/scipy/pull/18926/ :
 compare to https://github.com/scipy/scipy/pull/18926/files#diff-80d62cb9e2daea8b1e9d439cfcadeaf56bbf4dc8e41435f07a1d69089f5f875cR11

Reply:
That sounds fine, but why is `scipy/signal/_splines.py` not being removed in this commit then? (48dfec69357e1f0983426ddd76e295e077cfa452)